### ISSUE TITLE
Fortress and Void Thaumaturge armor values give more than usual

### ISSUE THREAD

**Issue by BIoxyest** [Azanor/thaumcraft-beta#1697]
Void Thaumaturge armor gives twice the amount than specifed on its tooltip, and Fortress armor does something similar but not quite. I have provided a video of the armors displaying this bug.
https://www.youtube.com/watch?v=jIugZu7Wo8w
and here is a gif of the void armor being equipped with the mods attributefix and overloadedarmorbar installed, which lets the armor value on the player go above 30 points and allows your armor bar to display over 20, respectively.
![void](https://user-images.githubusercontent.com/51765001/59389329-aa359e80-8d33-11e9-922b-dd16b78654b4.gif)

I know Fortress armor is supposed to increase its armor value a little bit when you wear more of them, but I dont think its supposed to give you +1 armor when only one piece is worn.

**Comment by Stalladir** [comment]
Not a bug, each piece of those sets resonates with each other increasing armor value. If I remember correctly it's even described in Fortress armor research.

**Comment by GrigLog** [comment]
Yeah, void robe's armor display is broken. It does not give you 40 defense points, only 20.
